packages/spl/package.json

```diff
@@ -1,11 +1,10 @@
 {
-  "private": true,
   "name": "@audius/spl",
-  "version": "0.0.10",
+  "version": "0.0.12",
   "description": "The core Typescript mappings to the Audius Solana Programs",
-  "main": "./dist/cjs/index.js",
```

Comment:
oh wow, the ./ excludes it?

Reply:
Apparently! I don't understand why, just noticed sdk was doing it this way